.gitignore

```diff
@@ -5,6 +5,9 @@ _temp/
 gulp-tsc-tmp-*
 .gulp-tsc-tmp-*
 
+shared-npm-packages/**/*.js
```

Comment:
Is there a reason we don't want to call these `common-npm-packages`? That shows the relation to what we had before but also that they are on npm now.

Reply:
Good call, updated to do this